WHITEPAPER.md

```diff
@@ -172,8 +172,8 @@ or move on to the next round. The proposer for a round is chosen
 deterministically from the ordered list of validators, in proportion to their
 voting power.
 
-The full details of the protocol are described
-[here](https://github.com/tendermint/tendermint/wiki/Byzantine-Consensus-Algorithm).
+For full details of the protocol, see  
```

Comment:
Instead of "here", let's use descriptive link text

https://developers.google.com/style/link-text